github/projects.go

```diff
@@ -65,6 +65,10 @@ type ProjectOptions struct {
 	Name string `json:"name,omitempty"`
 	// The body of the project. (Optional.)
 	Body string `json:"body,omitempty"`
+
+	// The following fields are only applicable for update.
```

Comment:
nit: It seems slightly odd to me to describe everything here in the plural when it is just a single field, but not a big deal.
another nit: we typically comment on the line before as to what values a field can have, but this is not a big deal either.

Reply:
Yeah a bit of an oversight by me, I've amended the commit to include those changes as I think they are valuable as well.

Reply:
Unlike Gerrit, on GitHub, I can’t see what has changed if you amend a commit and force push it. Can you tell me what you changed please?

Reply:
> another nit: we typically comment on the line before as to what values a field can have, but this is not a big deal either.

Again, it’s hard for me to be sure, but wasn’t this already the case before? The `Either "open" or "closed".` sentence. I wish I could see what changed... :(